source/request-as-event-emitter.js

```diff
@@ -47,13 +47,26 @@ module.exports = options => {
 
 		/* istanbul ignore next: electron.net is broken */
 		if (options.useElectronNet && process.versions.electron) {
-			const electron = global['require']('electron'); // eslint-disable-line dot-notation
+			const electron = require('electron');
```

Comment:
It was like this for a reason. It's to prevent Webpack from complaining. I guess we should add a comment about that.

Reply:
IDK why but the "global" approach was throwing an error for me. I'll try that again.

Reply:
This is what I get:
```js
(node:8174) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): TypeError: global.require is not a function
```

Reply:
Hmm... Smart :P